packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -1,13 +1,16 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
Any specific reason for this file-wide exception?

Reply:
will move it to concrete line. But yes, I bumped into case where it was not possible to satisfy this rule.